doc/modules/changes/20170510_anne-glerum

```diff
@@ -0,0 +1,6 @@
+Changed: The chunk geometry now removes the manifold id only from the boundaries and sets the correct
+boundary objects for dealii versions prior to version 9. For dealii 9, only the manifold is used,
```

Comment:
OK, I just wanted to check whether 9.0 really was the correct version number. Thanks for confirming.